### PR TITLE
Use e164 phone input from common

### PR DESCRIPTION
**Pull request by stefanfisk**

### CODE REVIEW

**Comment by stefanfisk** [comment]
Unlike in `www` this does not autofill the country prefix, because I couldn't see a quick and easy way of getting the orgs country code.

File: src/scss/_mixins.scss [hunk 1]
```diff
@@ -101,15 +101,34 @@
...
     font-weight: 500;
-    border-width: 0;
+    border-width: $border;
     display: inline-block;
     padding: 1.2em 1em 0.9em;
     border-bottom: 3px solid;
     border-radius: 3px;
-    @include button-color($color, $amount);
+    @if $transparent {
+            background-color: transparentize($color, 1);
```
Comment (richardolsson) [inline src/scss/_mixins.scss]: Indentation seems weird here.

Yes, yes I know, a linter would have caught this. We're working on it. 😜

**Comment by richardolsson** [comment]
Sorry for the base-branch-changing-spam.

Once approved, I will merge this into `feature/mass-sms` and continue to merge any features related to this into that branch, to be merged into master and released no sooner than R19.

File: src/scss/_mixins.scss (hunk 1 above)
Comment (stefanfisk) [reply src/scss/_mixins.scss]: Ah, yeah that was just a sloppy copy-paste on my part, pushed a fix!

**Pull request merged**